samples/bluetooth/iso_broadcast/src/main.c

```diff
@@ -133,7 +140,7 @@ void main(void)
 		net_buf_reserve(buf, BT_ISO_CHAN_SEND_RESERVE);
 		sys_put_le32(++iso_send_count, iso_data);
 		net_buf_add_mem(buf, iso_data, sizeof(iso_data));
-		ret = bt_iso_chan_send(&bis_iso_chan, buf);
+		ret = bt_iso_chan_send(&bis_iso_chan[0], buf);
```

Comment:
Consider putting the iso_send stuff in a for loop for each channel/stream

Reply:
done